icu4j/maven-build/README.md

```diff
@@ -118,3 +118,29 @@ where `<projectlist>` is a comma-separated list of names of the subfolders which
 If you want to run only a specific test(s), use the `-Dtest="<test>"` option, where `<test>` can be a test name, a class name / package prefix, or a comma-separate list of them.
 
 If you want to skip tests, use the `-DskipTests=true` option.
+
+#### Special Note about the Resources
+Currently, the Maven build reads the resources NOT from `icu4j/main/shared/data/icudata.jar` but from the individual files checked into [icu4j/maven-build/maven-icu4j-datafiles/src/main/resources/com/ibm/icu/impl/data/icudt73b](https://github.com/unicode-org/icu/tree/main/icu4j/maven-build/maven-icu4j-datafiles/src/main/resources/com/ibm/icu/impl/data/icudt73b) directory (when the version of ICU is 73). Therefore, if your PR changed any resource data files (ex: the ones contained within`icudata.jar`), you also need to update the corresponding resources files in order for your test to pass in the Maven build.
```

Comment:
You can also add that in the `icu4j/maven-build` folder there is a script named `extract-data-files.sh` that one can use for the update.

Reply:
And if you go there, maybe make the script executable, because it is not :-)
```
git update-index --chmod=+x extract-data-files.sh
```

Reply:
This does not look like resolved, as resolving it would change `extract-data-files.sh`, which is not part of this PR.
But this comment was not really related to updating the README, so I will approve as is.